pkg/config/appsync_server_config.go

```diff
@@ -55,10 +58,12 @@ func NewAppSyncConfig(ctx context.Context) (*AppSyncConfig, error) {
 }
 
 func (c *AppSyncConfig) Validate() error {
-	if url, err := url.Parse(c.AppSyncURL); err != nil {
-		return fmt.Errorf("unable to parse APP_SYNC_URL: %v", err)
-	} else if url == nil {
-		return errors.New("expecting a value for APP_SYNC_URL")
+	if c.AppSyncURL != "" {
```

Comment:
Nit: It'd be less nesting if we checked the negative case and returned `nil`:

```go
if c.AppSyncURL == "" {
  return nil
}

if url, err := ...
```

Reply:
I was worried if we added more validations later someone would miss that. Done.